src/System.IdentityModel.Tokens.Jwt/JwtSecurityTokenHandler.cs

```diff
@@ -675,12 +684,12 @@ private bool ValidateSignature(byte[] encodedBytes, byte[] signature, SecurityKe
         /// <param name="token">A 'JSON Web Token' (JWT) that has been encoded as a JSON object. May be signed 
         /// using 'JSON Web Signature' (JWS).</param>
         /// <param name="validationParameters"><see cref="TokenValidationParameters"/> that contains signing keys.</param>
-        /// <exception cref="ArgumentNullException"> thrown if 'token is null or whitespace.</exception>
-        /// <exception cref="ArgumentNullException"> thrown if 'validationParameters is null.</exception>
-        /// <exception cref="SecurityTokenValidationException"> thrown if a signature is not found and <see cref="TokenValidationParameters.RequireSignedTokens"/> is true.</exception>
-        /// <exception cref="SecurityTokenSignatureKeyNotFoundException"> thrown if the 'token' has a key identifier and none of the <see cref="SecurityKey"/>(s) provided result in a validated signature. 
+        /// <exception cref="ArgumentNullException">Thrown if 'token is null or whitespace.</exception>
```

Comment:
All the other exception descriptions start with "if", not with "thrown if". Can we make them consistent?